packages/core/src/transactions/TransactionPath.ts

```diff
@@ -0,0 +1,18 @@
+import TransactionRequest from './TransactionRequest'
+import App from '../wrappers/App'
+
+export interface TransactionPathData {
+  apps: App[]
+  destination: string
+  transactions: TransactionRequest[]
+}
+
+export default class TransactionPath {
+  readonly apps!: App[]
+  readonly destination!: string
```

Comment:
I change the `destination` to be a string instead of `App` (what the specification had). If I'm not wrong the destination could be any address.